osu.Game.Rulesets.Taiko/Skinning/TaikoLegacySkinTransformer.cs

```diff
@@ -81,11 +81,38 @@ public Drawable GetDrawableComponent(ISkinComponent component)
                         return new LegacyBarLine();
 
                     return null;
+
+                case TaikoSkinComponents.TaikoExplosionGood:
+                case TaikoSkinComponents.TaikoExplosionGreat:
+                case TaikoSkinComponents.TaikoExplosionMiss:
+
+                    var sprite = this.GetAnimation(getHitname(taikoComponent.Component), true, false);
+                    if (sprite != null)
+                        return new LegacyHitExplosion(sprite);
+
+                    return null;
             }
 
             return source.GetDrawableComponent(component);
         }
 
+        private string getHitname(TaikoSkinComponents component)
```

Comment:
nit, as usual
```suggestion
        private string getHitName(TaikoSkinComponents component)
```
(+ usages)